components/Typeahead.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  AsyncTypeahead,
+  AsyncTypeaheadProps,
+  Menu,
+  MenuItem,
+  TypeaheadModel,
+} from 'react-bootstrap-typeahead'; // Typeahead
+import 'react-bootstrap-typeahead/css/Typeahead.css'; //Typeahead styling
+import Helmet from 'react-helmet'; // Helmet
+import 'bootstrap/dist/css/bootstrap.min.css'; //Bootstrap styling
+import { Box, Center, Divider, VStack, Text } from '@chakra-ui/layout'; // Chakra UI
+
+// Typeahead props
+type Props<T extends TypeaheadModel> = Pick<
+  AsyncTypeaheadProps<T>,
+  'isLoading' | 'onSearch' | 'renderMenuItemChildren'
+> & {
+  results: Array<T>;
+  placeholder: string;
+};
+
+export default function Typeahead<T extends TypeaheadModel>(props: Required<Props<T>>) {
+  const { isLoading, onSearch, renderMenuItemChildren, results, placeholder } = props;
+  const filterBy = () => true;
+
+  return (
+    <>
+      <AsyncTypeahead
+        filterBy={filterBy}
+        id="async-typeahead"
+        isLoading={isLoading}
+        minLength={3}
+        onSearch={onSearch}
+        options={results}
+        placeholder={placeholder}
+        emptyLabel="No results found."
+        positionFixed={true}
+        inputProps={{
+          style: {
+            width: '466px',
+            height: '44px',
+          },
+        }}
+        renderMenuItemChildren={renderMenuItemChildren}
+        renderMenu={(results, menuProps, props) => {
+          menuProps.text = props.text;
+          return (
+            <VStack>
+              <Box></Box>
```

Comment:
Remove this?

Reply:
The`Box` adds a small vertical space between the input field and the menu drop down. I changed this to be a `Spacer` instead to make its purpose more evident